strings/c-or-cpp/count-words.c

```diff
@@ -0,0 +1,66 @@
+/*  Program to find the total numbers of words in a sentence  */
+#include<stdio.h>
+#include<string.h>
+#include<stdlib.h>
+
+//Function which calculate and return total numbers of words
+int count_words(char *sen)
+{
+    int i=0,check=1,word=0;
+    //dynamically allocating memory
+    realloc(sen,strlen(sen)+1);
```

Comment:
Return value of realloc is being ignored here. 

also there is no need for reallocation here as you have already done it one before( line 49 ).

Reply:
yah sorry didn't see it thanks for telling me